plugin/js/showpass-custom.js

```diff
@@ -212,8 +259,36 @@
 					params['tracking-id'] = Cookies.get('affiliate');
 				}
 
-				openShowpassWidget(slug, params);
+				openShowpassWidget(id, params, 'product');
+			});
 
+			$('body').on('click', 'a[href*="showpass.com"].force-showpass-widget', function(e) {
+				e.preventDefault();
+				
+				let slug = $(this).attr('href').split('.com/')[1];
```

Comment:
this parsing looks a bit sus not taking into account query params or hashes

Reply:
Yes the characters after the domain are always the slug so this is a safe assumption. And it doesn't look at query params but given that this functionality is not listed in the readme at all I'm inclined not to increase the scope to support this.

It seems like you need to be aware of the `force-showpass-widget` class to even have this work. @marc-thomas is there any information about which client this was added for and if we need to continue supporting it?